scripts/ci-e2e.sh

```diff
@@ -47,8 +47,8 @@ export GCP_NETWORK_NAME=${GCP_NETWORK_NAME:-"${TEST_NAME}-mynetwork"}
 GCP_B64ENCODED_CREDENTIALS=$(base64 "$GOOGLE_APPLICATION_CREDENTIALS" | tr -d '\n')
 export GCP_B64ENCODED_CREDENTIALS
 export KUBERNETES_MAJOR_VERSION="1"
-export KUBERNETES_MINOR_VERSION="22"
-export KUBERNETES_PATCH_VERSION="6"
+export KUBERNETES_MINOR_VERSION="23"
+export KUBERNETES_PATCH_VERSION="1"
```

Comment:
same: you can bump to 1.23.3, but I would keep that in 1.22.6 for now

Reply:
we dont have prebuit images for 1.23.1 only for 1.23.3, can you please update for that?

Reply:
Sure!